montecarlo/cfast_mcarlo.py

```diff
@@ -803,12 +803,16 @@ def _draw_doors_and_holes_opening(self):# {{{
                 how_much_open=binomial(1,vents[v_type])
                 self._psql_log_variable(v_type.lower(),how_much_open)
 
+            # only regular door can be burnt through
             if how_much_open == 0:
-                door['CRITERION'] = self.conf['doors_break']['criterion']
-                door['SETPOINT'] = self.conf['doors_break']['setpoint']
-                door['PRE_FRACTION'] = 0
-                door['POST_FRACTION'] = 1
-                door['DEVC_ID'] = f"t_{v['name']}"
+                if v_type == 'DOOR':
+                    door['CRITERION'] = self.conf['doors_break']['criterion']
+                    door['SETPOINT'] = self.conf['doors_break']['setpoint']
+                    door['PRE_FRACTION'] = 0
+                    door['POST_FRACTION'] = 1
+                    door['DEVC_ID'] = f"t_{v['name']}"
+                else:
+                    break
```

Comment:
else continue, otherwise the remaining doors will be skipped

Reply:
Agreed, pushing a commit with line 815 changed.